benchmark/type/find_if/brigand.list.cpp.erb

```diff
@@ -7,17 +7,15 @@
 #include <brigand/types/bool.hpp>
 #include <brigand/types/integer.hpp>
 
-struct is_last {
-    template<typename N>
-    using apply = brigand::bool_<N::type::value == <%= n %>>;
-};
+template<typename N>
+using is_last = brigand::bool_<N::type::value == <%= n %>>;
 
 using list = brigand::list<
     <%= (1..n).map { |i| "brigand::int32_t<#{i}>" }.join(', ') %>
 >;
 
 #if defined(METABENCH)
-using result = brigand::find<list, is_last>;
+using result = brigand::find<list, brigand::bind<is_last, brigand::_1>>;
```

Comment:
the algorithms themselves (replace_if, find, etc.) should have all the needed headers.